internal/util-position/src/test/scala/sbt/internal/util/SourcePositionSpec.scala

```diff
@@ -0,0 +1,18 @@
+package sbt.internal.util
+
+import org.scalatest._
+
+class SourcePositionSpec extends FlatSpec {
+  "SourcePosition()" should "return a sane SourcePosition" in {
+    val filename = "SourcePositionSpec.scala"
+    val lineNumber = 9
+    SourcePosition() match {
```

Comment:
This looks like a pure expression, but it actually side effects to the author's machine.
It would be good have another name to signal it's a macro.